Agents/AccessAgent/README.md

```diff
@@ -1,163 +1,151 @@
-# Introduction
+# Access Agent
 
-The purpose of the Access Agent is to provide a point of access to the knowledge graph for executing SPARQL query and update operations. The Access Agent hides the storage implementation layer, so that the user or calling agent does not need to know the store location or software e.g. Blazegraph, RDF4J etc.
+The Access Agent provides a point of access to the knowledge graph for executing SPARQL operations and finding the associated relational database. The Access Agent hides the storage implementation layer, so that the user or calling agent does not need to know the location or software e.g. Blazegraph, RDF4J etc. 
```

Comment:
I would suggest deleting the first sentence altogether, and move the second sentence between the first and second sentences of the "Purpose" subsection.